client/src/services/documentService.ts

```diff
@@ -0,0 +1,76 @@
+import axios from 'axios';
+
+declare var DATASTORE_URI: string;
+
+export const DocumentService = {
```

Comment:
Is there any reason to export this as a record? (I guess it does not matter, but some of our other code just exports functions, which might be simpler..)